slack/table_slack_user.go

```diff
@@ -88,8 +88,25 @@ func listUsers(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData)
 		plugin.Logger(ctx).Error("slack_user.listUsers", "connection_error", err)
 		return nil, err
 	}
-	// NOTE: This API does automatic paging
-	users, err := api.GetUsersContext(ctx)
+
```

Comment:
Hey @japborst , for the current function we're using, `GetUsersContext`, there seems to be a way to pass in `GetUsersOptions` directly to that function. Is it possible to pass in a limit there instead of handling the paging manually?

Reply:
Reference link to function definition - https://github.com/slack-go/slack/blob/master/users.go#L389

Reply:
Oh sorry, I had just read the results of you trying this exact approach in your original PR discussion.

Does the Slack Go SDK have any examples of recommendations on how to handle limits with the GetUsersContext function (which is what they seem to recommend using)?

Reply:
Unfortunately not. Was referring to this in the PR description. Unfortunately that only controls the page size, but doesn't limit. Looking at the implementation it'll keep on fetching and only halt upon error or when done.

Reply:
@cbruno10 sorry see your latest comment just now. I don't see any guidance from their docs, as that's primarily just autogenerated from the implementation.

Looking at their code, I also don't see any other function we could be using here.

One thing to check if if this indeed works well when no limit is set. Our Slack directory is so large that this never completes for me within the ratelimit boundaries.

Reply:
Hey @japborst, thanks for tackling this one!